py/server/deephaven/experimental/s3.py

```diff
@@ -118,12 +237,22 @@ def __init__(self,
                     (access_key_id is None and secret_access_key is not None)):
                 raise DHError("Either both access_key_id and secret_access_key must be provided or neither")
 
+            def throw_multiple_credentials_error(credentials1: str, credentials2: str):
+                raise DHError(f"Only one set of credentials can be set, but found {credentials1} and {credentials2}")
+
+            # Configure the credentials
```

Comment:
All of the deprecated inputs should be raising deprecation warnings.  @jmao-denver can give details.  This doc may be relevant.  https://dev.to/hckjck/python-deprecation-2mof

Reply:
We also need a ticket for next release to remove the deprecated stuff.

Reply:
Created this and added a TODO in the code: https://github.com/deephaven/deephaven-core/issues/6165